src/components/Shared/Feed.tsx

```diff
@@ -39,7 +48,7 @@ const List = ({ deleteMotion, deletePost, loading }: Props) => {
       </>
     );
 
-  return <CircularProgress />;
+  return mounted && <CircularProgress />;
```

Comment:
I think this is what's causing the type error in `pages/index`. The following should resolve it:

```
  if (mounted) <CircularProgress />;
  return <></>;
```

Reply:
This works and resolves the error in index!